### ISSUE TITLE
Inclusive searching

### ISSUE THREAD

**Issue by RaccoonHours** [KuroZen/r34#11]
**Describe the feature**
Hey! Thanks so much for your reply to my other request!! I had one other request that im not sure if its possible but would love to see♡ would it be possible to have the option to make tags inclusive? An example is that the alot of malexmale stuff appears in different tags (male only, male/male, yaoi, ect) so would there be a way to allow it to search for images that have either tag rather than both? I know this must be asking for alot ahahaha and if It's not possible I totally understand!! Thank you for your time!

**Comment by kurozenzen** [comment]
Hi again.

You are way quicker than I am at responding.

The feature you are describing is something I wanted to have for a long time. Last time I checked it was completely broken on rule34.xxx though. It should be supported now from what I can tell. (See https://rule34.xxx/index.php?page=help&topic=cheatsheet). I will look into it.

Also, if you have any other suggestions or similar I would prefer if you put them on the new github (https://github.com/kurozenzen/r34-react/issues). They are easier to find there.

See ya 👋

**Comment by kurozenzen** [comment]
Yup. This is very much doable. I just have to come up with a good way to do the user interface for it.

**Comment by kurozenzen** [comment]
DONE. Its a bit experimental atm. I am open to suggestions on how to improve it.

Clicking the Modifier button (the plus in the tag search) cycles through PLUS (posts must include this tag), MINUS (post must not include this tag) and WAVE (posts must include one out of all the WAVE tags).

Have fun!
